### ISSUE TITLE
v-(select or highlight)-on-focus

### ISSUE THREAD

**Issue by ShGKme** [syropian/vue-select-on-focus#1]
Hello.

In `README`:

```
# Usage
## Globally

import Vue from "vue";
import vSelectOnFocus from "vue-select-on-focus";

Vue.use(vSelectOnFocus);
```
Then `select-on-focus` is been mentioned many times in README.

But if I use `vue-select-on-focus` globally, directive `select-on-focus` doesn't work. 

After looking to the src I found that default name is `highlight-on-focus`, not `select-on-focus`.

I think, this fact should be mentioned in README or (better) default directive name should be changed to `select-on-focus`.

**Comment by syropian** [comment]
Whoops, good catch, I originally went with `highlight-on-focus` for the whole lib before renaming and I must have forgot to change the reference there. I will get this fixed soon!

**Comment by syropian** [comment]
Fixed in 2.0.0
